src/Themes/Profile.scss

```
@@ -0,0 +1,51 @@
+.backgroundHolder {
```

Use kebab-case for naming class.
```suggestion
.background-holder {
```